From 3577a6471ff63d3fee823f758e37985546a163f4 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Mon, 30 Nov 2020 20:21:31 +0200
Subject: [PATCH] [SW-20154] goya2: add header header guards for common.h

Add a header guards to prevent multiple inclusions

Change the license header to fix the checkpatch
warning -
WARNING: Misplaced SPDX-License-Identifier tag - use line 1 instead

Change-Id: I3c2c99b480d4b0f28f71dc54b1423e30a12ad3fa
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/goya2/common.h b/goya2/common.h
index 0df56d6..36c1f56 100644
--- a/goya2/common.h
+++ b/goya2/common.h
@@ -1,9 +1,10 @@
-/*
- * (C) Copyright 2019 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
+/* SPDX-License-Identifier:     GPL-2.0+
  *
- * SPDX-License-Identifier:     GPL-2.0+
+ * Copyright (C) 2020 HabanaLabs Ltd.
  */
+
+#ifndef COMMON_GOYA2_H_
+#define COMMON_GOYA2_H_
 
 #include <preboot.h>
 #include <goya2_blocks_linux_driver.h>
@@ -82,3 +83,5 @@
 
 #define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_MODE_G3_CONF	0x5555
 #define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_SEL_G4_CONF	0xFF
+
+#endif /* COMMON_GOYA2_H_ */
diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index 49e56cb..f037021 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -4,6 +4,9 @@
  * All Rights Reserved.
  */
 
+#ifndef COMMON_GOYA2_ARC_H_
+#define COMMON_GOYA2_ARC_H_
+
 #include <preboot.h>
 #include <goya2_blocks_linux_driver.h>
 
@@ -42,3 +45,5 @@
  */
 #define EXTRACT_BITS(value, pos, num) ((value & (((1 << num) - 1) << pos))\
 						>> pos)
+
+#endif /* COMMON_GOYA2_ARC_H_ */
diff --git a/goya2_arc_gnu/common.h b/goya2_arc_gnu/common.h
index f70fc39..447d013 100644
--- a/goya2_arc_gnu/common.h
+++ b/goya2_arc_gnu/common.h
@@ -1,9 +1,10 @@
-/*
- * (C) Copyright 2019 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
+/* SPDX-License-Identifier:     GPL-2.0+
  *
- * SPDX-License-Identifier:     GPL-2.0+
+ * Copyright (C) 2020 HabanaLabs Ltd.
  */
+
+#ifndef COMMON_GOYA2_ARC_GNU_H_
+#define COMMON_GOYA2_ARC_GNU_H_
 
 #include <preboot.h>
 #include <goya2_blocks_linux_driver.h>
@@ -34,3 +35,5 @@
  */
 #define EXTRACT_BITS(value, pos, num) ((value & (((1 << num) - 1) << pos))\
 						>> pos)
+
+#endif /* COMMON_GOYA2_ARC_GNU_H_ */
